### PR TITLE
initialzed with bower.json and package.json 

### PR DESCRIPTION
**Pull request by fobabett**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (keokilee) [inline bower.json]: Not sure you guys will need a bower.json quite yet. Can remove this for now.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fobabett/api-server.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fobabett/api-server/issues"
```
Comment (keokilee) [inline package.json]: Change these URLs to reference hitraffic/api-server.

**Comment by fobabett** [comment]
changed links

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hitraffic/api-server.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/hitraffic/api-server/issues"
+  },
+  "homepage": "https://github.com/fobabett/api-server"
```
Comment (keokilee) [inline package.json]: Oops, just missed this last one.

**Comment by keokilee** [comment]
giphy

**Pull request merged**